### ISSUE TITLE
[BUG] File > Export as PNG makes the preview blink when file has many layers

### ISSUE THREAD

**Issue by xan2622** [photopea/photopea#4790]
**Describe the bug:**

"File > Export as PNG" makes the "preview frame" blink.

**Steps to reproduce the behavior:**
1. Create or open a file with many layers (6 in my case)
2. Click on "File > Export as PNG"
3. At this moment, you might see the "preview frame" blinking

**Expected behavior:**

A "preview frame" not blinking.

**Screenshots:**

![firefox_1zh8CDt8yG](https://user-images.githubusercontent.com/1732773/177029978-3c1e18a9-4969-4a9c-b6d9-018ea34b180c.gif)

**System Information:**
 - OS: Windows11
 - Browser version: Firefox 102.0

**Additional context:**

After merging all the 6 layers and retrying to Export as PNG, this doesn't happen anymore.

**Comment by photopea** [comment]
Did you name your layers with _a_ at the beginning? That activates the animation mechanism: https://www.photopea.com/learn/animations

**Comment by xan2622** [comment]
> Why did you make a video with all the workspace except the list of layers?

No real reason. But you are right, I shouldn't have cut this area out of my previous screenshot.

![Screenshot 2022-07-03 131636](https://user-images.githubusercontent.com/1732773/177037070-a6b7fbad-29b7-40aa-9f2a-906a4c2e8c11.png)

> Did you name your layers with _a_ at the beginning? That activates the animation mechanism: https://www.photopea.com/learn/animations

Good find. I didn't rename these layers myself. At first, I just loaded a GIF file, Photopea loaded all frames (each one in a different layer), and indeed with `_a_` at the beginning. Then, I only kept one layer, I started editing that layer, I duplicated it, then added other layers on top of them, etc.  Then, when I finished this "project" I wanted to export it as a still PNG image (not an animation).

Now I understand that the preview blinked because of these two layers: 

![Screenshot 2022-07-03 133311](https://user-images.githubusercontent.com/1732773/177037678-1a0a8b4f-6960-4fa7-9241-8664d39a3649.png)
